portals/devportal/source/src/app/components/Shared/AppsAndKeys/AppConfiguration.jsx

```diff
@@ -81,7 +81,7 @@ const AppConfiguration = (props) => {
         classes, config, isUserOwner, previousValue, handleChange,
     } = props;
 
-    const [selectedValue, setSelectedValue] = useState(previousValue);
+    let [selectedValue, setSelectedValue] = useState(previousValue);
```

Comment:
Let's use const, Let allow altering the variable which we don't do for state values or setter function

Reply:
Fixed with 531cac593c47ce021eeb12d46186a3a97b480c00